script/build.ts

```diff
@@ -326,6 +326,23 @@ function copyDependencies() {
     cp.execSync('yarn install', { cwd: outRoot, env: process.env })
   }
 
+  console.log('  Copying desktop-trampoline…')
+  const desktopTrampolineDir = path.resolve(outRoot, 'desktop-trampoline')
+  const desktopTrampolineFile =
+    process.platform === 'win32'
+      ? 'desktop-trampoline.exe'
+      : 'desktop-trampoline'
+  fs.removeSync(desktopTrampolineDir)
+  fs.mkdirSync(desktopTrampolineDir)
+  fs.copySync(
+    path.resolve(
+      projectRoot,
+      'app/node_modules/desktop-trampoline/build/Release',
+      desktopTrampolineFile
+    ),
+    path.resolve(desktopTrampolineDir, desktopTrampolineFile)
+  )
```

Comment:
This snippet here makes sure the desktop-trampoline binary is copied from the `node_modules` folder to the app bundle.

Reply:
Will the exe binary need to be signed on Windows in order to be operational?

Reply:
Not sure if it's necessary, but definitely something we want (it's required on macOS). It's in my TODO list for tomorrow 😄

Reply:
FYI @mislav I can confirm the new binary is signed for both macOS and Windows without any extra effort 😌 🎉